### ISSUE TITLE
About mimic explainer:

### ISSUE THREAD

**Issue by GISH123** [interpretml/interpret-community#311]
Sorry, not sure if its appropriate to ask questions via issues, but I see no other ways of doing it.

As my understanding, Mimic explainer is kind of like LIME, but using 4 : LightGBM , Linear Regression , Stochastic Gradient Descent , and Decision Tree  as surrogate model.

My question is, is it appropriate to use Mimic's LightGBM to explain my LightGBM model?(does it make sense?)
Or another way to put it, how should i explain my LightGBM model?

And is there any way to learn more about Mimic explainer? like research paper documents or something.
Thank you!

**Comment by gaugup** [comment]
Hi @GISH123,

Sorry for the late response. If you are using the LightGBM model, then hopefully you will have "feature_importances_" that can give you ordering of the features in your trained lightgbm model.

However, the explainers under mimic explainer also give you local feature importance per sample of your evaluation data. So you could train lightgbm model and explain it using the mimic explainer explainer with lightgbm as the surrogate model to get both local and global feature importance. This is a perfectly valid scenario. However, the surrogate model might be lossy but for the scenario that you described the explanations provided by lightgbm surrogate model for your lightgbm model should be very close. 

You can read more about the blackbox explainers and surrogate models on the link https://christophm.github.io/interpretable-ml-book/global.html.

Let us know if you have more questions.

Regards,

**Comment by gaugup** [comment]
@GISH123, I hope my above response helped with your understanding of blackbox explainers. Sicen there is no response on this issue for sometime, I am closing this issue. IF you have further questions, please feel free to open another issue.

Regards
